client/scripts/index.js

```diff
@@ -20,6 +22,20 @@ function getPlayerColorFromNumber(player_number) {
 
 }
 
+function enableButtons(data){
```

Comment:
why does this take the argument data ?

Reply:
this also seems to modify the whiteboard
soo
maybe call the function
enableButtonsAndBoard